lib/matplotlib/axis.py

```diff
@@ -774,6 +774,11 @@ def _set_scale(self, value, **kwargs):
         self.isDefault_majfmt = True
         self.isDefault_minfmt = True
 
+    def _get_shared_axes_siblings(self):
```

Comment:
Seems more consistent to call this _get_shared_axes (consistently with _get_shared_axis)?  Also perhaps move the definition of _get_shared_axis just below this one as they belong logically together.